src/app/home/home.page.html

```diff
@@ -34,94 +34,92 @@
       </ion-select>
     </ion-item>
     <ion-item>
-      <ion-icon id="prevDay" name="chevron-back-outline" (click)="decrementDate()" class="icon"></ion-icon>
+      <ion-icon id="prevDay" name="chevron-back-outline" (click)="handleBack()" class="icon"></ion-icon>
       <ion-label type="date" class="select">
         <h2>{{this.formattedDate}} &nbsp;</h2>
         <ion-button (click)="today()" id="today" fill="outline">Heute</ion-button>
       </ion-label>
-      <ion-icon id="nextDay" class="button" name="chevron-forward-outline" (click)="incrementDate()" class="icon"></ion-icon>
+      <ion-icon id="nextDay" class="button" name="chevron-forward-outline" (click)="handleNext()" class="icon"></ion-icon>
     </ion-item>
   </ion-list>
-  <div class="menu-container" id="menu-container">
-    <ng-template [ngIf]="!this.loading" [ngIfElse]="showLoading">
-      <div class="no_meals" *ngIf="this.currentMeals.length === 0">
-        <p>Leider hat die Mensa an diesem Tag nichts für dich zu bieten...</p>
-        <img src="assets/hungry.png" alt="Kein Essen" />
-      </div>
```

Comment:
Hast du das beabsichtigt rausgenommen?

<ng-template [ngIf]="!this.loading" [ngIfElse]="showLoading">
      <div class="no_meals" *ngIf="this.currentMeals.length === 0">
        <p>Leider hat die Mensa an diesem Tag nichts für dich zu bieten...</p>
        <img src="assets/hungry.png" alt="Kein Essen" />
      </div>

Reply:
nope